content/learn.html

```diff
@@ -13,10 +13,10 @@
   <div class="container">
     <div class="hero-area hero-area-row row">
       <h1 class="hero-title no-link">Learn</h1>
-      <p>Explore our structured learning paths to discover everything you need to know about building for the modern web.</p>
+      <p>For each topic, learn best practices for the modern web and hone your skills with a hands-on codelab.</p>
```

Comment:
```suggestion
      <p>Learn best practices for the modern web and hone your skills with hands-on codelabs.</p>
```

Reply:
ok I think this comment should now show a button for you to acceept or commit the suggestion I just made. If you click that I think it'll update for you.